index.js

```diff
@@ -13,6 +13,12 @@ export function render ({ props, state }, setState) {
   const { items, onChange, removeOnHide } = props;
   const { activeTabIndex } = state;
 
+  if (process.browser) {
+    window.addEventListener('deku-tab-control:setState', function (e) {
+      setState({ activeTabIndex: e.detail.activeTab });
+    });
+  }
```

Comment:
hm, this does not seem like the right way to do this. It's tricky because we're using setState here and it's not playing well with also using props.